packages/gravity-ui-web/src/sass/04-objects/_layouts.scss

```diff
@@ -0,0 +1,28 @@
+// some utility classes to use the grav-l-column-layout mixin directly as classes with sensible breakpoints.
+.grav-o-two-column {
+  @media (min-width: grav-breakpoint(small)) {
+    @include grav-l-column-layout(2);
+  }
+}
+
+.grav-o-up-to-four-column {
```

Comment:
Hmm... I think having a consistent naming scheme between these classes would be nice. Either both should be `.grav-o-up-to-X-column` or both should be `.grav-o-X-column`. I'm leaning towards the latter, as its more concise. In any case, I think the documentation in the "n-column" markdown file should explain that these classes are always "up to N columns from the widest breakpoint onwards, and reduce to lower number of columns on narrower viewports".

Also, if we're adding 2 and 4 column classes, should we not add a 3 col one for completeness?